R/plots-gi.R

```diff
@@ -1 +1,121 @@
 # Genetic interaction calculation step related plots can go here
+
+library(tidyverse)
```

Comment:
This should be added as an @import to the pertaining functions instead of this. This way this library is only loaded if it is needed.